src/olympia/bandwagon/views.py

```diff
@@ -138,11 +139,38 @@ def get_object(self):
             self.lookup_field = '%s__slug' % self.lookup_field
         return super().get_object()
 
+    @classmethod
+    def _transformer(self, objs):
+        current_versions = [
+            obj.addon._current_version for obj in objs
+            if obj.addon._current_version
+        ]
+        addons = [obj.addon for obj in objs]
+        Version.transformer_promoted(current_versions)
+        Version.transformer_license(current_versions)
+        attach_tags(addons)
+
+    @classmethod
+    def _locales_transformer(self, objs):
+        current_versions = [
+            obj.addon._current_version for obj in objs
+            if obj.addon._current_version
+        ]
+        addons = [obj.addon for obj in objs]
+        attach_trans_dict(CollectionAddon, objs)
+        attach_trans_dict(Addon, addons)
+        attach_trans_dict(License, [ver.license for ver in current_versions])
+
     def get_queryset(self):
         qs = (
             CollectionAddon.objects
             .filter(collection=self.get_collection())
-            .prefetch_related('addon'))
+            .prefetch_related('addon__promotedaddon', 'addon__addon_tags')
```

Comment:
Why do you need `addon__addon_tags` here, shouldn't it be handled by your call to `attach_tags` in the `_transformer` ?

Reply:
you're right, that's one more query down https://github.com/mozilla/addons-server/pull/16082/commits/aa96ec8673353d9bc86611d02f2c5679167cf3fe